library/src/main/java/nl/surfnet/nonweb/sso/SSOServiceActivity.java

```diff
@@ -62,29 +48,50 @@ public void onCreate(Bundle savedInstanceState) {
         if (BuildConfig.DEBUG) {
             Log.d(TAG, "Starting task to retrieve token.");
         }
-        new RequestTokenTask(this, _consumerId).execute();
+        new RequestTokenTask(this, //
```

Comment:
No need to put the parameters in multiple lines if it wouldn't exceed line length